packages/store/ts/modules.d.ts

```diff
@@ -0,0 +1 @@
+declare module "prettier-plugin-solidity";
```

Comment:
I think this shouldn't be needed in `store` and `world`, doing this in just `common` should be sufficient, as the other packages don't even directly install this plugin (at least locally for me it seems to work)